### ISSUE TITLE
Conflict with bootstrap

### ISSUE THREAD

**Issue by EvaRune** [pixelunion/Extended-Tumblr-Photoset#19]
Just a suggestion to change .row class to something else to avoid conflicts with bootstrap or other frameworks using this class.

**Comment by graygilmore** [comment]
You're right, we should be using either `.pxu-row` or make it an option you can pass in like `.pxu-photo`.
